sql/database.sql

```diff
@@ -8519,11 +8533,12 @@ CREATE TABLE `uuid_mapping` (
   `id` bigint(20) NOT NULL AUTO_INCREMENT,
   `uuid` binary(16) NOT NULL DEFAULT '',
   `resource` varchar(255) NOT NULL DEFAULT '',
+  `resource_path` VARCHAR(255) DEFAULT NULL,
   `table` varchar(255) NOT NULL DEFAULT '',
   `target_uuid` binary(16) NOT NULL DEFAULT '',
   `created` timestamp NULL,
   PRIMARY KEY (`id`),
-  KEY `uuid` (`uuid`),
+  UNIQUE KEY `uuid` (`uuid`),
```

Comment:
Any advantage to this?
Purposefully made it not unique for use case of one uuid to represent a group of target uuid (that use case does not exist, but figured it would at some point)

Reply:
My thought is to make it unique so we don't inadvertently create data integrity issues.  If we want one uuid to represent a group of uuids we can remove the unique constraint at that point.

Reply:
sounds reasonable. then should also add this change to the upgrade script and update the following comment (to tune of if need to map one uuid to a group of uuids, then need to remove the unique contraint):
https://github.com/openemr/openemr/blob/master/src/Common/Uuid/UuidMapping.php#L23-L24

Reply:
actually, i don't know if we have a way to do this in the upgrade script (we can check if a key exists, but pretty sure can't check if a key is unique etc.).

Reply:
if we don't have a way to add to upgrade (and then a way to add unique key in future when needed), then may be best to let this sit as is (ie. not unique).

Reply:
K, I'll remove the unique constraint

Reply:
K, just pushed this fix.